src/components/PublishArticle.jsx

```diff
@@ -0,0 +1,129 @@
+import React, { useEffect, useState } from "react";
+import axios from "axios";
+import Preview from "./Preview";
+import { Container, Grid, Form, Dropdown, Button } from "semantic-ui-react";
+
+const PublishArticle = (props) => {
+  const [selectedArticle, setSelectedArticle] = useState();
+  const [message, setMessage] = useState("");
+  const [radio, setRadio] = useState("free");
+  let headers = JSON.parse(localStorage.getItem("J-tockAuth-Storage"));
+  headers = {
+    ...headers,
+    "Content-type": "application/json",
+    Accept: "application/json",
+  };
+
+  const cats = [
+    "Other",
+    "Sport",
+    "Local",
+    "Politics",
+    "Economy",
+    "World",
+    "Entertainment",
+  ];
+
+  const categories = cats.map((cat) => {
+    return { key: cat, text: cat, value: cat.toLowerCase() };
+  });
+
+  useEffect(() => {
+    const fetchSelectedArticle = async () => {
```

Comment:
This is already done inside the review component.
Either render this as a child of the review component and pass it down, or extract it to a module.

Reply:
Extracted